content/pop-incomplete-upgrade.md

```diff
@@ -22,10 +22,9 @@ section: software-troubleshooting
 tableOfContents: true
 ---
 
-
 ## Repairing Upgrade Errors
 
-Due to the overwhelming demand for 21.10, we have had to expand our servers' bandwidth. If you are experiencing any connection errors please run the following commands in a terminal.
+Due to the overwhelming demand for Pop!\_OS, users may occasionally experience bandwidth issues. If you are still experiencing any connection errors after waiting a short time (15-30 mins.), please run the following commands in a terminal.
```

Comment:
The period followed by a comma around the parenthesis looks weird. I would recommend just spelling out "minutes" to avoid that issue. (That also goes for this sentence in `upgrade-pop.md`.)

Reply:
Fixed with [bc0c77a](https://github.com/system76/docs/pull/998/commits/bc0c77a6b04b71d92e784ad4c065cffd1033f4b3).